docs/user-guide/gaffer-basics/what-is-gaffer.md

```diff
@@ -0,0 +1,24 @@
+# What is Gaffer?
+
+Gaffer is a graph database framework, it acts similarly to an interface providing a graph data
+structure on top of a chosen storage technology to enable storage of large graphs and traversal of
+it's nodes and edges. In a nutshell Gaffer allows you to take data, convert it into a graph, store it
+in a database and then run queries and analytics on it.
+
+The high level interactions of loading data and querying are demonstrated in the diagrams below.
+
+```mermaid
+flowchart TD
+    subgraph Graph Query
+        E((User)) --> F
+        G{{Schema}} --> F
+        F([Query]) ---> H(Gaffer)
+        J(key-value store) <--> H
+        H --> K([Result])
+    end
+    subgraph Data Input
+        A(Data)-->B{{Schema}}
+        B --> C(Gaffer)
+        C --> D(key-value store)
+    end
+```
```

Comment:
I don't understand this. The query looks like it takes a user and schema as input?

Reply:
This diagram was from the existing getting started guide, the query is like that as 99% of the time you need to know the schema to formulate a decent query on the graph e.g. what properties an edge might have etc.

Reply:
That's true. But to me, that diagram makes it seem like a query has a `User` and a `Schema` as inputs. I think further annotation of the links to explain the relationships would solve this.

Reply:
That's fair maybe something to address under the work to revamp that "what is gaffer?" page

Reply:
Yes, would be good to take a note and address under an issue for that.